moonshot/src/api/api_red_teaming.py

```diff
@@ -0,0 +1,36 @@
+import asyncio
+
+from moonshot.src.redteaming.attack.attack_module import AttackModule
+from moonshot.src.runners.runner import Runner
+
+
+# ------------------------------------------------------------------------------
+# Red teaming APIs
+# ------------------------------------------------------------------------------
+def api_run_red_teaming(runner: Runner, rt_args: dict) -> None:
```

Comment:
currently mine only have it under runner. run_recipes and stuff, i didnt put this under api,

Reply:
ok not sure how this will be called but i created an api so it can be called to run red teaming directly